paho/client.go

```diff
@@ -52,7 +52,6 @@ type (
 		// caCtx is used for synchronously handling the connect/connack
```

Comment:
Should fix up this comment if caCtx is going away

Reply:
Good catch, fixed :+1: